bigbluebutton-html5/private/config/settings.yml

```diff
@@ -295,3 +295,22 @@ private:
     - DoLatencyTracerMsg
   serverLog:
     level: info
+  minBrowserVersions:
+    - browser: chrome
+      version: 59
+    - browser: firefox
+      version: 52
+    - browser: firefoxMobile
+      version: 52
+    - browser: edge
+      version: 17
+    - browser: ie
+      version: Infinity
```

Comment:
Infinity means not supported?

Reply:
Yeah. Functionally it means that every version that is checked will fail.

Reply:
What is the difference between listing a browser with Infinity version and not listing the browser at all (i.e. not whitelisting any versions)?

Reply:
That I don't know. I copied what was in the example from the release notes. Also, in this case it wouldn't matter because I'm pretty sure there are other parts that also set the minimum version to Infinity.